### PR TITLE
Update version tested against to 20.5.0

### PR DESCRIPTION
**Pull request by omejia10**
- [X ] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/nimi-python/blob/master/CONTRIBUTING.md).

- [X] I've updated [CHANGELOG.md](https://github.com/ni/nimi-python/blob/master/CHANGELOG.md) if applicable.

~- [ ] I've added tests applicable for this pull request~

### What does this Pull Request accomplish?

Update "Driver Version Tested Against" to 20.5.0:

- 20.5.0 Driver Runtimes has been installed on nimi-bot
- Update applies to NI-DCPower, NI-SWITCH, and NI-DMM
### List issues fixed by this Pull Request below, if any.

none

### What testing has been done?

CI will run the tests.

### CODE REVIEW

**Comment by sbethur** [comment]
test this please

File: CHANGELOG.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ All notable changes to this project will be documented in this file.
     * #### Added
     * #### Changed
         * Fix [#1491](https://github.com/ni/nimi-python/issues/1491): import_attribute_configuration_buffer() fails intermittently when `list` or `array.array` is passed in.
+        * Update "Driver Version Tested Against", in documentation, with latest versions installed on nimi-bot. The version is 20.5.0 for NI-DCPower, NI-SWITCH, and NI-DMM. no changes on other drivers
```
Comment (marcoskirsch) [inline CHANGELOG.md:32]: Do we always call this out? It seems to be irrelevant to the CHANGELOG.md.
If we do always call it out, then keep it.

File: README.rst [hunk 2]
```diff
@@ -304,7 +304,7 @@ NI Switch Executive Python API Status
 +-------------------------------+---------------------+
 | NI Switch Executive (nise)    |                     |
 +===============================+=====================+
-| Driver Version Tested Against | 20.0.0              |
+| Driver Version Tested Against | 20.5.0              |
```
Comment (marcoskirsch) [inline README.rst:307]: You said:

> Update "Driver Version Tested Against" to 20.5.0:
> 
> 20.5.0 Driver Runtimes has been installed on nimi-bot
> Update applies to NI-DCPower, NI-SWITCH, and NI-DMM, whose latest released Runtime is 20.5.0

*  NI Switch Executive is not NI-SWITCH.
* Reword description to be less redundant.

**Review by marcoskirsch — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (marcoskirsch) [reply CHANGELOG.md:32]: I suppose it's fine, it tells the client that we have already tested it against these versions. What do you think Shreyas?

Comment (sbethur) [reply CHANGELOG.md:32]: It seems unnecessary, but you could also argue it's technically correct since we are changing user-facing documentation. Also, we have done it many times in the past. I'd say we keep it, don't think anyone will complain about us providing this extra info.

**Review by sbethur — CHANGES_REQUESTED** [review]

File: README.rst (hunk 2 above)
Comment (omejia10) [reply README.rst:307]: This was a mistake! I should have changed NI-SWITCH.

**Review by marcoskirsch — APPROVED** [review]

**Review by sbethur — APPROVED** [review]
